docs/networks/l16-testnet.md

```diff
@@ -181,3 +191,53 @@ make log_validator
 # You can stop the validator using, this will also stop all other nodes
 make stop
 ```
+
+# Troubleshooting in LUKSO Beta Testnet
+
+### 1. Permission denied while spinning up the node
+**Context**: While running `make start` you are getting permission related issues. You can have log like this:
+```
+Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/json": dial unix /var/run/docker.sock: connect: permission denied "docker kill" requires at least 1 argument. See 'docker kill --hel
+```
+**Proposed Solution:** try running make command as `super user`. For example: `sudo make start`
+
+### 2. Consensus (prysm) not syncing and Execution (geth) stops syncing after a few blocks
+**Context**: You found your consensus client has no peer and execution engine stops syncing after a few blocks
+**Proposed Solution:**
+1. Open `.env` file using any text editor. For `vim` the command will be `vim .env`
+2. Change `PRYSM_BOOTSTRAP_NODE` to this `PRYSM_BOOTSTRAP_NODE=enr:-MK4QACsMyCBqoH7E2xTFMyVKd0wbaOEoff6q_N1Vx_HVZuVYBk1JoB5Ava9h6eBlS5XzxM5LHFI1BG1IchMdI6JMhWGAX8tHtE1h2F0dG5ldHOIAAAAAAAAAACEZXRoMpC3QoawYgAAcf__________gmlkgnY0gmlwhCJbPjCJc2VjcDI1NmsxoQJp3RTwCXObnrJNuiJlLaM4LlhYOaWXhtj4Hz3PW9sfgYhzeW5jbmV0cwCDdGNwgjLIg3VkcIIu4A`
+3. restart the node using: `sudo make stop && sudo make start`
+
+### 3. For Ubuntu 20.04 LTS you get unmarshal related issue:
+**Context:**  Check your execution log by `sudo make log_execution`. If you find this:
+```
+log_execution: err="peer connected on snap without compatible eth support" log_consensus: level=error msg="Could not connect to powchain endpoint: could not dial eth1 nodes: json: cannot unmarshal string into Go struct field SyncProgress.CurrentBlock of type uint64" prefix=powchain
+```
+**Proposed solution**:
+```
+# stop docker containser
+sudo make stop
+# reset data directory
+sudo make reset
+# remove previous images
+docker system prune --all --force --volumes
+# delete lukso testnet directory
+cd .. && rm -rf ./lukso-l16-testnet
+```
+Then follow the doc and re-run everything from the start.
+
+
+# FAQ:
```

Comment:
I suggest you put it as Header 2

so it looks like this:

<img width="277" alt="image" src="https://user-images.githubusercontent.com/477945/158546889-9478b748-a78b-49a8-b0f3-2c0df49835b5.png">


instead of this

<img width="297" alt="image" src="https://user-images.githubusercontent.com/477945/158546936-7ca36e2b-5d7d-410b-80fa-2c312e1d5e72.png">


in the right sidebar